3ds Max/Max2Babylon/Forms/ExporterForm.Designer.cs

```diff
@@ -93,6 +93,7 @@ private void InitializeComponent()
             this.exportOptionsScrollPanel = new System.Windows.Forms.Panel();
             this.logTabPage = new System.Windows.Forms.TabPage();
             this.pictureBox1 = new System.Windows.Forms.PictureBox();
+            this.webBrowser1 = new System.Windows.Forms.WebBrowser();
```

Comment:
What needed to be changed in the exporter form? I can see alot of changes to some element positioning, and a new WebBrowser component, is this needed?

Reply:
my mistake, nothing to change here